lib/decryption.rb

```diff
@@ -0,0 +1,41 @@
+class Decryption
+  attr_reader :character_set
+
+  def initialize
+    @character_set = (("a".."z").to_a << " ").reverse
+  end
+
+  def indexed_alphabet
```

Comment:
Pretty sure every other method in this class is ~~ identical to the functionality in encryption. AKA use case for module or inheritance!!!!